src/main/java/uk/gov/hmcts/reform/sscs/service/venue/VenueRpcDetailsService.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.sscs.service.venue;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
+import uk.gov.hmcts.reform.sscs.model.venue.VenueRpcDetails;
+import uk.gov.hmcts.reform.sscs.service.VenueDataLoader;
+
+import java.util.Objects;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isYes;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class VenueRpcDetailsService {
```

Comment:
Funny thing is.. I'm creating a service pretty much identical to this.. it's in bulk-scan just now but moving it across to common soon.

https://github.com/hmcts/sscs-bulk-scan/pull/1260/files#diff-34bfc460a64fff7abdf4a6afe4b0b4363fd15d4b74d836151d3f6cd7db1fa9c0

https://github.com/hmcts/sscs-common/pull/1196/files#diff-e4edc919f586317d7f0e19dbe482d8d8cb5ae2bc0bc7294512823d36c3f78240

Perhaps we can merge them at some point?

Reply:
I have move these to commons:
https://github.com/hmcts/sscs-common/commit/02b480d2b46c8cba84c0d79afdc888c319807af7